.github/workflows/pr.yaml

```
@@ -62,12 +68,16 @@ jobs:
           fi
 
   prevent-merge-with-lean-ci:
+    permissions:
+      contents: read
     runs-on: ubuntu-latest
     needs: check-lean-ci
     steps:
       - name: Check lean-ci status
+        env:
+          LEAN_CI: ${{ steps.check-lean-ci.outputs.lean_ci_enabled }}
```

actionlint (1.7.12)

[error] property "check-lean-ci" is not defined in object type {}

(expression)